manifests/istio-control/istio-autoinject/templates/mutatingwebhook.yaml

```diff
@@ -1,5 +1,5 @@
 {{- $ca := genCA "istio-sidecar-injector-ca-{{ .Release.Namespace }}" 3650 }}
-{{- if not .Values.global.operatorManageWebhooks }}
+{{- if and (not .Values.global.operatorManageWebhooks) (not .Values.global.istiod.enabled) }}
```

Comment:
Why is this changing? Bad merge?

Reply:
I had to fix this manually - it's not related to this PR but a test is failing here that I think was passing elsewhere because of a race. Without this change, we can have two webhook configs that race.